### DISCUSSION TITLE
Compartment fire reaction throughout entire domain.

### DISCUSSION THREAD

**Discussion by JesseSlaten** [firemodels/fds#13102] [Q&A] [answered]
Hello FDS'ers,

I am attempting at modeling a compartment fire simulation of a toxic gas release (burning PVC to create HCL), larger context of modeling the fire in under-ventilated conditions. 

To start what I have been doing is preparing a small slab of PVC and trying to 'recreate' the Verification/Species/pvc_combuston.fds reaction, which likely is the cause of my issue. Goal in mind is to be able to track the concentration of HCL produced in a series of experiments with varying ventilation.  What I am seeing is the entire compartment is burning, while my external flux is relativley high (EXTERNAL_FLUX=75.), am I accidentally igniting all the air in the domain ? 

![wpifire_0002](https://github.com/firemodels/fds/assets/25551081/1840c7a7-be94-4af9-8688-a4a9273306ab)

Attached as well is my .fds file, (there's some stuff commented out that my aim is to create the 'compartment' which controls the ventilation for our test sample), I have changed the filename as github doesn't seem to like the .fds extension in markdown.

[wpifireE2257.txt](https://github.com/user-attachments/files/15991346/wpifireE2257.txt)

Thanks, Jesse.

**Comment by rmcdermo** [comment, marked as answer]
I don't think you want this line:
```
&INIT MASS_FRACTION(1)=0.229,SPEC_ID(1)='PVC' /
```

**Reply by JesseSlaten** [reply]
Ran a quick simulation without that line, looks like that was it. 
![wpifire_0022](https://github.com/firemodels/fds/assets/25551081/68ac2892-7f7e-469a-be1c-0e2e1b0dadcb)


If there's no HCL now can I instead use VOLUME_FRACTION(1) = A NUMBER  ?

Thanks!
